front-end/src/pages/home/home.tsx

```diff
@@ -66,8 +66,8 @@ export default function Home() {
           <AngleBox light={theme.palette.secondary.main} dark={theme.palette.secondary.dark} theme={{ dark: theme.palette.primary.main, light: theme.palette.info.light }} sx={{ backgroundColor: theme.palette.secondary.main, color: theme.palette.info.light }} onClick={() => goTo('search')}>
             <Typography fontFamily="Montserrat" variant="h3">PLAYER STATS</Typography>
           </AngleBox>
-          <AngleBox light={theme.palette.websiteBackground.main} dark={theme.palette.websiteBackground.dark} theme={theme.palette.primary.main} sx={{ backgroundColor: 'black', color: theme.palette.info.light }} onClick={() => goTo('discord')}>
-            <Typography fontFamily="Montserrat" variant="h3">DISCORD</Typography>
+          <AngleBox light={theme.palette.websiteBackground.main} dark={theme.palette.websiteBackground.dark} theme={theme.palette.primary.main} sx={{ backgroundColor: 'black', color: theme.palette.info.light }} onClick={() => goTo('leagues')}>
+            <Typography fontFamily="Montserrat" variant="h3">LEAGUE STATS</Typography>
```

Comment:
w0w deleting the discord link zzz

Reply:
My assumption is that 99% of players get to this site from the discord so it feels redundant LOL